src/effects/RealtimeEffectList.h

```diff
@@ -25,18 +25,19 @@ class RealtimeEffectState;
 
 class Track;
 
-struct RealtimeEffectListMessage
+struct RealtimeEffectListMessage final
 {
    enum class Type
    {
-      Insert,///<New effect item was added to the list at srcIndex position
-      Replace,///<Effect item was replaced with a new item at srcIndex position
-      Remove,///<Effect item was removed from the list at srcIndex position
-      Move ///<Item position has changed, from srcIndex to dstIndex
+      Insert,///<New effect item was added to the list at srcIndex position. affectedState is a new state
+      Replace,///<Effect item was replaced with a new item at srcIndex position. affectedState is an old state.
+      Remove,///<Effect item was removed from the list at srcIndex position. affectedState is removed state.
+      Move ///<Item position has changed, from srcIndex to dstIndex.  affectedState is the moved state
```

Comment:
This comment is not consistent with `RealtimeEffectList::MoveEffect` which leaves the pointer in the message null